utils/toml/toml.go

```diff
@@ -0,0 +1,43 @@
+package toml
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pelletier/go-toml/v2"
+)
+
```

Comment:
Would it make sense to add a some unit tests for these?

Reply:
added